mem-reserve

```diff
@@ -208,11 +224,64 @@ def parse_size(s):
                 value *= 1024
         return value
 
+def add_suffix(s):
+
+        p_val = 1024 * 1024 * 1024 * 1024 * 1024
```

Comment:
p_val = 1<<50
t_val = 1 << 40
g_val = 1<<30
etc

Reply:
Was trying to match the existing parse_size() routine style, probably slower to do multiplication, but maybe more readable?